### DISCUSSION TITLE
[suggestion] Social spy for mcMMO party chat?

### DISCUSSION THREAD

**Discussion by mrfloris** [Zrips/CMI#1338] [Enhancement]
Would it be possible to get partychat socialspy for mcMMO 2.1?

this way the community helpers can keep an eye on abuse there, and won't depend on the owner to review console logs.

**Comment by TheWranger** [comment]
Do you mean that CMI socialspy should hook into mcMMO and display messages from party chat?

**Comment by mrfloris** [comment]
> Do you mean that CMI socialspy should hook into mcMMO and display messages from party chat?

Yeah basically.
